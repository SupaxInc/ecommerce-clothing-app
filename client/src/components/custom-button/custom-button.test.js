import React from "react";
import { shallow, mount } from "enzyme";

import * as redux from 'react-redux';
import CustomButton from "./custom-button";

describe('Custom Buttom component', () => {
    let wrapper;

    afterEach(() => {
        wrapper = shallow(<CustomButton />);
    });

    it('should render custom button',() => {
        expect(wrapper).toMatchSnapshot();
    });
});