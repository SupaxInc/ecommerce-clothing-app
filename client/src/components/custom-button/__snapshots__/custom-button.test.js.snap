// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom Buttom component should render custom button 1`] = `undefined`;
